src/exe/colmap.cc

```diff
@@ -266,6 +266,7 @@ int RunStereoFuser(int argc, char** argv) {
   std::string workspace_format = "COLMAP";
   std::string pmvs_option_name = "option-all";
   std::string output_path;
+  std::string output_type;
```

Comment:
Can we make the previous "ply" the default output type? This also requires to change to AddDefaultOption below. Thanks.

Reply:
no problem